source/server-discovery-and-monitoring/tests/integration/isMaster-command-error.yml

```diff
@@ -27,7 +27,7 @@ tests:
       appname: commandErrorHandshakeTest
     operations:
       # The command error on the initial handshake should mark the server
-      # Unknown and clear the pool.
+      # Unknown (emitting a ServerDescriptionChangedEvent) and clear the pool.
```

Comment:
Related to my question in slack. I initially thought this would not emit a ServerDescriptionChangedEvent until I realized that a ServerDescription with error != ServerDescription without error.